ra2ce/graph/exporters/geodataframe_network_exporter.py

```diff
@@ -30,7 +30,7 @@
 
 class GeoDataFrameNetworkExporter(NetworkExporterBase):
     def export_to_shp(self, output_dir: Path, export_data: gpd.GeoDataFrame) -> None:
```

Comment:
This should be renamed to `export_to_gpkg`

Reply:
Thanks! I missed a few instances. Now it should be good to go.